react/AppLinker/index.jsx

```diff
@@ -49,12 +53,20 @@ export class AppLinker extends React.Component {
     }
   }
 
-  static getOnClickHref(props, nativeAppIsAvailable) {
+  getOnClickHref(props, nativeAppIsAvailable) {
```

Comment:
It should be kept static please. git blame is your friend: https://github.com/cozy/cozy-ui/commit/653068575bcc189123c2700817bea579a4bd7096